js/src/index.js

```diff
@@ -54,16 +54,15 @@ if (process.env.NODE_ENV === 'development') {
 }
 
 const AUTH_HASH = '#/auth?';
-const parityUrl = process.env.PARITY_URL || window.location.host;
-const urlScheme = window.location.href.match(/^https/) ? 'wss://' : 'ws://';
+const parityUrl = process.env.PARITY_URL || '127.0.0.1:8546';
 
 let token = null;
 
 if (window.location.hash && window.location.hash.indexOf(AUTH_HASH) === 0) {
   token = qs.parse(window.location.hash.substr(AUTH_HASH.length)).token;
 }
 
-const api = new SecureApi(`${urlScheme}${parityUrl}`, token);
+const api = new SecureApi(parityUrl, token);
```

Comment:
Ok, I missed a step although I did see the stuff below. However with line 57 above now, we only do localhost and cannot connect to anything that is on the network?